src/imageio/format/jpeg.c

```diff
@@ -379,7 +379,7 @@ int write_image(dt_imageio_module_data_t *jpg_tmp, const char *filename, const v
     }
   }
 
-  uint8_t *row = malloc((size_t)3 * jpg->global.width * sizeof(uint8_t));
```

Comment:
This can't be the fix as those are only for jpeg export.

Reply:
Well, actually that's probably a fix for the out buffer whereas mine is for the input one. All in all, I cannot get a crash on jpg with this PR and my fix.

Reply:
it never hurts to align properly anyway